src/Bot.php

```diff
@@ -342,6 +342,11 @@ protected function processPlugins(array $plugins, array $processors)
         foreach ($processors as $processor) {
             foreach ($plugins as $plugin) {
                 $processor->process($plugin, $this);
+                if ($plugin instanceof PluginContainerInterface) {
+                    foreach ($plugin->getPlugins() as $childPlugin) {
```

Comment:
Instead of looping over the child plugins here, simply recursively invoke `$this->processPlugins()` like so:

```php
$this->processPlugins($plugin->getPlugins(), $processors);
```

This will allow for plugin processors to run on nested plugin containers.